soc/riscv/riscv-ite/it8xxx2/Kconfig.defconfig.series

```diff
@@ -16,6 +16,9 @@ config ITE_IT8XXX2_INTC
 config RISCV_GP
 	default y
 
+config RISCV_MUL_ISA
```

Comment:
Is there an errata somewhere? It would be nice to add a link to it, for future reference.

Reply:
I’m collecting errata at the same time, will get back to you.

Reply:
@edersondisouza Added the errata link. Please revisit.